client/my-sites/themes/thanks-modal.jsx

```diff
@@ -44,6 +44,10 @@ const ThanksModal = React.createClass( {
 		page( this.props.site.URL );
 	},
 
+	visitSiteDisabled() {
+		return this.props.hasActivated ? false : true;
```

Comment:
Can't we just 

```js
return ! this.props.hasActivated;
```

(and maybe even inline that expression instead of having a method for it?)